aries-site/src/examples/templates/forms/SignUpExample.js

```diff
@@ -81,17 +82,28 @@ const FormContainer = ({ ...rest }) => {
 };
 
 export const SignUpExample = () => {
+  const themeMode = useDarkMode().value ? 'dark' : 'light';
   const [formValues, setFormValues] = React.useState({
     email: 'jane.smith@hpe.com',
   });
+  const [backgroundError, setBackgroundError] = React.useState('');
   const [passwordRules, setPasswordRules] = React.useState(passwordRulesStrong);
+  //  Since password rules are being checked outside the validation
+  //  and being passed in the info message the background color is not
+  //  being changed. We need to check if all password rules are applied
+  //  then we can change background color to indicate there is no error.
 
   const onChange = values => {
     setFormValues(values);
     const adjustedPasswordRules = passwordRules.map(rule => {
       const adjustedRule = { ...rule };
       const valid = adjustedRule.regexp.test(values.password);
       adjustedRule.valid = valid;
+      if (valid !== true) {
+        setBackgroundError(themeMode === 'dark' ? '#C54E4B5C' : '#FC61613D');
```

Comment:
Do we even need to call for themeMode anymore? Could we just set it to `validation-critical` and grommet would take care of the dark or light mode?

Reply:
no this will change